content/docs/command-reference/exp/init.md

```diff
@@ -19,43 +18,57 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 
 ## Description
```

Comment:
Somewhere in the description, it might be useful to explain that `dvc exp init` by default expects that input data, parameters, and source code paths exist before running an experiment, and that the command is expected to generate models, metrics, and plots.

Reply:
That's more or a problem for `exp run` though @dberenbaum right (already linked form `--run` option). But should we say that `exp run` (and `repro`) for that matter expect that the stage definition and code are valid and working (resp.)? Ideally it should be evident IMO.

Reply:
> That's more or a problem for `exp run` though

Well that's the point of `exp init`, right? Better to have users understand what's needed up front than to have them run `exp init` only to fail on `exp run`.

Doesn't need to be part of this PR. It could also be handled by some of the suggested changes to the core command rather than the docs.

Reply:
> Better to have users understand what's needed up front than to have them run exp init only to fail on exp run.

I'm not sure. If users need to read that on the cmd ref., is `exp init` serving as a self-explanatory command? I think those notes should be in the command's output fist (and added here as a result of that change).